network/src/main/scala/no/ndla/network/tapir/TapirServer.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Part of NDLA network
+ * Copyright (C) 2023 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.network.tapir
+
+import cats.effect.IO
+import net.bull.javamelody.{MonitoringFilter, ReportServlet}
+import org.http4s.HttpApp
+import org.http4s.jetty.server.JettyBuilder
+import org.log4s.{Logger, getLogger}
+
+import javax.servlet.DispatcherType
+import scala.concurrent.Future
+
+case class TapirServer(name: String, serverPort: Int, app: HttpApp[IO], enableMelody: Boolean)(onReady: => Unit = {}) {
+  val logger: Logger      = getLogger
+  private var serverReady = false
+
+  private def setupMelody(b: JettyBuilder[IO]): JettyBuilder[IO] = {
+    val filt          = new MonitoringFilter()
+    val dispatches    = java.util.EnumSet.of(DispatcherType.REQUEST, DispatcherType.ASYNC)
+    val reportServlet = new ReportServlet
+
+    b
```

Comment:
Liker at navn er beskrivende
```suggestion
    builder
```

Reply:
Om ikkje dette krasjer med builder rett under…